src/display/svg.js

```diff
@@ -321,91 +322,101 @@ var SVGExtraState = (function SVGExtraStateClosure() {
     this.maskId = '';
   }
 
-  SVGExtraState.prototype = {
-    clone: function SVGExtraState_clone() {
-      return Object.create(this);
-    },
-    setCurrentPoint: function SVGExtraState_setCurrentPoint(x, y) {
-      this.x = x;
-      this.y = y;
-    },
-  };
-  return SVGExtraState;
-})();
-
-SVGGraphics = (function SVGGraphicsClosure() {
-  function opListToTree(opList) {
-    var opTree = [];
-    var tmp = [];
-    var opListLen = opList.length;
-
-    for (var x = 0; x < opListLen; x++) {
-      if (opList[x].fn === 'save') {
-        opTree.push({ 'fnId': 92, 'fn': 'group', 'items': [], });
-        tmp.push(opTree);
-        opTree = opTree[opTree.length - 1].items;
-        continue;
-      }
+  clone() {
+    return Object.create(this);
+  }
 
-      if (opList[x].fn === 'restore') {
-        opTree = tmp.pop();
-      } else {
-        opTree.push(opList[x]);
-      }
-    }
-    return opTree;
+  setCurrentPoint(x, y) {
+    this.x = x;
+    this.y = y;
   }
+}
 
-  /**
-   * Formats float number.
-   * @param value {number} number to format.
-   * @returns {string}
-   */
-  function pf(value) {
-    if (Number.isInteger(value)) {
-      return value.toString();
+// eslint-disable-next-line no-inner-declarations
```

Comment:
This is required (here and below) because these are now declared inside an `if` statement inside of in a closure like before (the `if` statement being the one that excludes all SVG code from non-generic builds).

Reply:
The objective is to remove the `if` statement around all SVG code later on if the SVG back-end is more stable. The only reason it's excluded from non-generic builds is because it was experimental and not yet used.